gcloud/bigtable/table.py

```diff
@@ -139,6 +139,36 @@ def create(self, initial_split_keys=None):
         # We expect a `._generated.bigtable_table_data_pb2.Table`
         client._table_stub.CreateTable(request_pb, client.timeout_seconds)
 
+    def rename(self, new_table_id):
+        """Rename this table.
+
+        .. note::
+
+            This cannot be used to move tables between clusters,
+            zones, or projects.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+                ``BigtableTableService.RenameTable is not yet implemented``
+
+            when this method is used. It's unclear when this method will
+            actually be supported by the API.
```

Comment:
```python
>>> try:
...     TABLE.rename('gcloud-pythoo-new-name')
... except Exception as exc:
...     pass
...
>>> exc
grpc.framework.interfaces.face.face.LocalError()
>>> exc.code
<StatusCode.UNIMPLEMENTED: 12>
>>> exc.details
'BigtableTableService.RenameTable is not yet implemented'
```

----

@nathanielmanistaatgoogle What is the difference between a `NetworkError` and `LocalError`?